core/templates/components/button-directives/hint-and-solution-buttons.component.spec.ts

```diff
@@ -59,14 +61,15 @@ describe('HintAndSolutionButtonsComponent', () => {
         {
           provide: TranslateService,
           useClass: MockTranslateService
-        }
+        }, contextService
       ]
     }).compileComponents();
   }));
 
   beforeEach(() => {
     fixture = TestBed.createComponent(HintAndSolutionButtonsComponent);
     component = fixture.componentInstance;
+    contextService = TestBed.inject(ContextService);
```

Comment:
@DhairyaMajmudar Why do you need to add context service? I can't see where you use it.